store/CHANGELOG.md

```diff
@@ -41,6 +41,10 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 * [#16321](https://github.com/cosmos/cosmos-sdk/pull/16321) QueryInterface defines its own request and response types instead of relying on comet/abci & returns an error
 
+### Bug Fixes
+
+* [#]() Propogate the Snapshotter's failure to the caller, (it will create a empty snapshot silently before).
```

Comment:
```suggestion
* [#16588](https://github.com/cosmos/cosmos-sdk/pull/16588) Propogate the Snapshotter's failure to the caller, (it will create a empty snapshot silently before).
```